cypress/Shared/MeasuresPage.ts

```diff
@@ -14,11 +14,21 @@ export class MeasuresPage {
     public static readonly paginationLimitEquals25 = '[data-value="25"]'
 
     //Measure Version
+    public static readonly versionMeasuresRadioButton = '[name="type"]'
+    public static readonly VersionDraftMsgs = '[data-testid="success-toast"]'
+    public static readonly VersionDraftErrMsgs = '[data-testid="error-toast"]'
+    public static readonly updateDraftedMeasuresTextBox = '[data-testid="measure-name-input"]'
+    public static readonly createDraftContinueBtn = '[data-testid="create-draft-continue-button"]'
     public static readonly measureVersionMajor = '[data-testid="radio-button-group"] > :nth-child(1)'
     public static readonly measureVersionContinueBtn = '[data-testid="create-version-continue-button"]'
     public static readonly measureVersionSuccessMsg = '[data-testid="success-toast"]'
     public static readonly measureVersioningErrorMsg = '[data-testid="error-toast"]'
+    public static readonly measureSelectBtn = '#measure-landing > div > div > div > div:nth-child(1) > div > div > div > div:nth-child(1) > table > tbody > tr > td:nth-child(4) > button'
+    public static readonly measureSelectContainer = '[class="btn-container" ]'
 
+    //<div class="btn-container"><button data-testid="view-measure-63d40acb1632e36a0f9f9653">View</button><button data-testid="create-version-measure-63d40acb1632e36a0f9f9653">Version</button></div>
+
```

Comment:
not sure if we need these comments

Reply:
removed all un-necessary comments from the MeasurePage file